lib/templates/fragments/menu.php

```diff
@@ -29,68 +34,85 @@ function beans_primary_menu() {
 		 *
 		 * @param array $args Nav menu arguments.
 		 */
-		$args = apply_filters( 'beans_primary_menu_args', array(
-			'theme_location' => has_nav_menu( 'primary' ) ? 'primary' : '',
-			'fallback_cb'    => 'beans_no_menu_notice',
-			'container'      => '',
-			'menu_class'     => $nav_visibility, // Automatically escaped.
-			'echo'           => false,
-			'beans_type'     => 'navbar',
-		) );
+		$args = apply_filters(
+			'beans_primary_menu_args',
+			array(
+				'theme_location' => has_nav_menu( 'primary' ) ? 'primary' : '',
+				'fallback_cb'    => 'beans_no_menu_notice',
+				'container'      => '',
+				'menu_class'     => $nav_visibility, // Automatically escaped.
+				'echo'           => false,
+				'beans_type'     => 'navbar',
+			)
+		);
 
 		// Navigation.
 		beans_output_e( 'beans_primary_menu', wp_nav_menu( $args ) );
 
 	beans_close_markup_e( 'beans_primary_menu', 'nav' );
-
 }
 
 beans_add_smart_action( 'beans_primary_menu_append_markup', 'beans_primary_menu_offcanvas_button', 5 );
 /**
  * Echo primary menu offcanvas button.
  *
  * @since 1.0.0
+ *
+ * @return void
  */
 function beans_primary_menu_offcanvas_button() {
 
 	if ( ! current_theme_supports( 'offcanvas-menu' ) ) {
 		return;
 	}
 
-	beans_open_markup_e( 'beans_primary_menu_offcanvas_button', 'a', array(
-		'href'              => '#offcanvas_menu',
-		'class'             => 'uk-button uk-hidden-large',
-		'data-uk-offcanvas' => '',
-	) );
-
-		beans_open_markup_e( 'beans_primary_menu_offcanvas_button_icon', 'i', array(
-			'class' => 'uk-icon-navicon uk-margin-small-right',
-		) );
+	beans_open_markup_e(
+		'beans_primary_menu_offcanvas_button',
+		'a',
+		array(
+			'href'              => '#offcanvas_menu',
+			'class'             => 'uk-button uk-hidden-large',
+			'data-uk-offcanvas' => '',
+		)
+	);
+
+		beans_open_markup_e(
+			'beans_primary_menu_offcanvas_button_icon',
+			'i',
+			array(
+				'class' => 'uk-icon-navicon uk-margin-small-right',
+			)
+		);
```

Comment:
This is another one that _could_ have all of the arguments inline. Why? The array only has one element in it.